packages/xo-server/src/xo-mixins/api.mjs

```diff
@@ -433,26 +432,36 @@ export default class Api {
         }).on('error', error => {
           captureLogs(undefined, () => log.warn(error))
         })
+
+        let success = true
         try {
-          result = await captureLogs(({ time, level, namespace, message, data }) => {
+          result = await captureLogs((log, fallbackTransport) => {
+            fallbackTransport(log)
```

Comment:
can it fail ?

Reply:
I guess it could but it should not, if our logger fail we will have bigger problems.